client-side/src/stories/verticalTabs/verticalTabss.jsx

```diff
@@ -0,0 +1,42 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { useTranslation } from 'react-i18next';
+import './verticalTabs.scss'
+const  VerticalTabs=({ labels=['tab1','tab2'],elements=[<p>tab 1</p>, <pp>tab 2</pp>]})=> {
```

Comment:
1. your props are required, so you dont have to write default values
2. notice that the name of the props will be same as in propTypes